src/ethereum/shanghai/transactions.py

```diff
@@ -20,8 +20,8 @@
 from .fork_types import Address
 
 TX_BASE_COST = 21000
-TX_DATA_COST_PER_NON_ZERO = 16
-TX_DATA_COST_PER_ZERO = 4
+TOTAL_COST_FLOOR_PER_TOKEN = 12
+STANDARD_TOKEN_COST = 4
```

Comment:
```suggestion
TOTAL_COST_FLOOR_PER_CALLDATA_TOKEN = 12
CALLDATA_TOKEN_COST = 4
```

Unless there's a compelling reason to say "STANDARD" here, I'd prefer not including it.

How do you feel about making it explicit these are calldata related constants?